bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,242 @@
+adjust these if you're running a node with limited memory resources.
+
+2. On `config.toml`, populate your seed nodes using entries from the 
+[network page](../README.md#seed-nodes):
+
+```shell
+# P2P Configuration Options    
+
+# This is only an example and should be replaced with actual testnet seed
+# nodes.
+# Comma separated list of seed nodes to connect to
+seeds = "8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656" 
+```
+
+Parameters:
+- `seeds`: Comma separated list of seed nodes that your node will connect to for 
+discovering other peers in the network
+
+Next, you'll need to obtain the network's genesis file. This file contains 
+the initial state of the blockchain and is crucial for successfully syncing 
+your node. You can inspect the file [here](../README.md#genesis) or use the 
+following commands to download it directly:
+
+```shell
+wget https://github.com/babylonlabs-io/networks/raw/main/bbn-test-5/genesis.tar.bz2 
+tar -xjf genesis.tar.bz2 && rm genesis.tar.bz2
+mv genesis.json <path>/config/genesis.json # You must insert the home directory of your node
+```
+
+## 3. Prepare for sync
+Before starting your node sync, it's important to note that the initial release 
+at genesis was `v0.9.0`, while subsequently there have been software upgrades.
+
+There are three options you can choose from when syncing:
+1. Sync through a network snapshot (fastest method)
+2. Sync through state sync (quick catch-up without full history)
+3. Sync from scratch (complete sync from block 1)
+
+### 3.1. Sync through a network snapshot
+
+Snapshot syncing is the fastest way to get your node up to date with the network. 
+A snapshot is a compressed backup of the blockchain data taken at a specific 
+height. Instead of processing all blocks from the beginning, you can download 
+and import this snapshot to quickly reach a recent block height.
+
+You can obtain the network snapshot [here](../README.md).
+
+<!-- TODO: We can add other snapshot sources as they appear -->
+To extract the snapshot, utilize the following command:
+
+```shell
+tar -xvf bbn-test-5.tar.gz -C <path>
+```
+
+Parameters:
+- `bbn-test-5.tar.gz`: Name of the compressed blockchain snapshot file
+- `<path>` : Your node's home directory
+
+After importing the state, you can now start your node as specified in section
+[Start the node](#4-start-the-node).
+
+### 3.2. Sync through state sync
+
+State sync downloads only the current blockchain state (account balances, 
+validator set, and module states) instead of processing the entire chain history.
+While this means you won't have historical data, state sync allows your node to 
+quickly catch up to the current state without downloading and verifying the 
+entire blockchain history. To find the state-sync server from our 
+[networks homepage](../README.md).
+
+To utilize state sync, you'll need to update a few flags in your `config.toml`:
+```shell
+[statesync]
+enable = true
+
+rpc_servers = "Y"
```

Comment:
Should we include the details of acquiring `trust_height` and `trust_hash` here or in the `networks hompage`? 
While the `rpc_servers` can be found in the network page, commands to get the  `trust_height` and `trust_hash` can be put here